src/applications/gi/components/vet-tec/VetTecCalculator.jsx

```diff
@@ -140,24 +145,48 @@ export class VetTecCalculator extends React.Component {
           (Learn more)
         </button>
       </div>
+      {// PROD FLAG CT 116 STORY 19868
+      environment.isProduction() ? (
+        <div>
+          <div className="row calculator-result">
+            <div className="small-8 columns">
+              <div>In person rate:</div>
+            </div>
+            <div className="small-4 columns vads-u-text-align--right small-screen:vads-u-padding-left--7">
```

Comment:
This long className is used four times. Might want to define a constant.

Reply:
Broke it out into separate variable